.github/ISSUE_TEMPLATE/design_or_architecture.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Software Design
+about: 
+title: 'Design ...'
+labels: 'T-design'
+assignees: ''
+---
+
+## Scope 
+<!-- Clarify what the design or architecture should cover (e.g., a system, feature, component, or entire product). -->
+
+## Functional Requirements
+<!-- A list of functional specifications the system or feature must fulfill. Use cases or user stories demonstrating expected behavior. -->
+
+## Non-Functional Requirements
+<!-- If applicable provide Performance, Scalability, Security or other metrics and characteristics of module to design -->
+
+
+## Related resources
+<!-- List resources that might be useful during design/research -->
+
+
+## Acceptance criteria
+<!-- Acceptance criteria in terms of design or architecture usually (but not only) should consist of:
+- list of open questions that need to be answered 
+- some summary, research paper or meta knowledge as an outcome of research or investigation
```

Comment:
Why is this in the design template as well as the investigation template? Didn't we decide to separate them?

Reply:
it is a design template. Refactored AC (I messed up then initially)